crates/astria-sequencer-relayer/src/finalization_pipeline/head_candidate.rs

```diff
@@ -0,0 +1,81 @@
+use astria_sequencer_types::SequencerBlockData;
+use tendermint::Hash;
+use tendermint_rpc::endpoint::block;
+
+#[derive(Clone, Default, Debug)]
+pub(crate) enum HeadCandidate {
```

Comment:
what's the purpose of this enum?

Reply:
to be able to make the separation between blocks proposed by this validator which will be submitted to da upon finalisation and blocks that are simply added to the pipeline to verify canonicity